pkg/cache/clusterqueue_test.go

```diff
@@ -94,309 +94,6 @@ func TestClusterQueueUpdateWithFlavors(t *testing.T) {
 	}
 }
 
-func TestFitInCohort(t *testing.T) {
```

Comment:
I can try to salvage these tests, if you think they're worth keeping. Let me know what you think, reviewers

Reply:
@mimowo